src/odemis/gui/cont/acquisition.py

```diff
@@ -742,7 +742,7 @@ def on_acquisition(self, evt):
         self._tab_panel.Layout()  # to put the gauge at the right place
 
         # start acquisition + connect events to callback
-        self.acq_future = acq.acquire(self._tab_data_model.acquisitionStreams)
+        self.acq_future = acq.acquire(self._tab_data_model.acquisitionStreams, self._main_data_model.md_manager)
```

Comment:
You also need to update the code for the SECOM. See the other callers of acq.acquire (I think it's in win.acquisition).
The plugins should also be updated. That's a bit of a pain. (and maybe it's a sign we should have merged the MetadataManager into a large "acquisition manager"... but might be better done later, in a refactoring task)